docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,6 +12,127 @@ keywords:
+1. The identified technical writer cuts a branch and creeates a PR with an empty What's New doc that will be populated with the What’s New content for the next release. 
+1. The technical writer then pins a link to [version-number-release-gtm](https://raintank-corp.slack.com/archives/C02L8GQJGBW) Slack channel.
+
+   > **Note:** A Product Manager (PM) is responsible for update the version number of that channel after a release goes out. 
+   
+1. The technical writer posts in the Slack channel that the What's New PR is ready for populating. 
+1. The PM and Product Marketing Manager (PMM) organizations identify which PM or PMM is responsible for vetting content added to the What's New draft.
+1. The PM or PMM on point communicates with eng squads without a PM resource about who’s on point for What’s New PR reviews from within the PM and PMM orgs.
+
+### Throughout a release
+
+A Grafana release cycle is typically 6-8 weeks, and during that time, developers are empowered to add to the What's New document incrementally. To see a list of release dates and assigned PMs and PMMs, refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG).
+
+During this time, the role of the technical writer is to periodically check in on the What's New PR to determine how much content has been added.
+
+Complete the following steps throughout a release cycle:
+
+1. Dev teams cut PRs against the branch and add What’s New content.
+   - If a dev team has a PM, the dev team adds that PM or PMM as Reviewer.
+   - If a dev team has a writer, the dev teams add the writer as Reviewer.
+   - If a dev team does not have a PM or PMM, the dev team adds the identified PM or PMM as the reviewer.
+1. Contributors label PRs with `add to what’s new` and write the PR description so it can be used as draft source for the What's New document. 
+
+   > **Note:** This won't need to apply to every PR - just the primary PR that checks in the new feature, changed feature, or deprecation. 
+
+   The following image shows an example What's New PR description field that include the information you should complete.
+
+   <figure>
+   <img src="whats-new-pr-template-example.png" alt="What's New PR template example" width="600">
+   </figure>
+
+   The following image show a completed description for a What's New PR.
+
+    <figure>
+   <img src="whats-new-completed-pr-example.png" alt="What's New completed PR" width="600">
+   </figure>
+
+   The following example shows a completed PR with labels applied: https://github.com/grafana/grafana/pull/56225
+
+1. PMs and/or PMMs review the content, offer suggestions, ask for clarifications, and generally collaborate.
+1. During the PM/PMM-to-dev collaboration cycle, technical writers collaborate to provide development and copy edit.
+1. After the review, the PM and/or PMM and technical writer approve the PR.
```

Comment:
@mjseaman , we can update these steps based on how we want to handle encouraging folks to cut PRs against the branch. These steps generally have not been happening due to direct commits.